docs/architecture/rpc.md

```diff
@@ -0,0 +1,26 @@
+---
+title: The RPC protocol
+authors: "Tim McMackin"
+lastUpdated: 26 October 2023
+---
+
+The Tezos RPC (Remote Procedure Call) protocol is a specification for a REST API that clients use to interact with Tezos nodes and nodes use to communicate with each other.
+Tezos nodes act as servers and accept HTTP requests from clients and other nodes via this protocol.
+
+- Clients use it to submit transactions and get information about the state of the blockchain, such as account balances and contract storage
+- Nodes use it to bake blocks and communicate with other nodes
```

Comment:
are you sure that node use RPCs to bake blocks? 
And they communicate with other nodes using RPCs? (normally, they use the P2P protocol)

Maybe you mean to the fact that baker daemons use RPCs to bake blocks and to communicate with the node?